lib/lightning/auditing.ex

```diff
@@ -8,9 +8,20 @@ defmodule Lightning.Auditing do
 
   def list_all(params \\ %{}) do
     from(a in Lightning.Credentials.Audit,
-      preload: [:actor],
       order_by: [desc: a.inserted_at]
     )
     |> Repo.paginate(params)
+    |> try_to_get_actors()
+  end
+
+  defp try_to_get_actors(page) do
+    page
+    |> Map.put(
+      :entries,
+      Enum.map(page.entries, fn entry ->
+        entry
+        |> Map.put(:actor, Repo.get(Lightning.Accounts.User, entry.actor_id))
+      end)
+    )
```

Comment:
I don't think we need to remove the preload, we can do a `left_join` to the users table and then preload on that join instead.

Also we could just not have a constraint/fk on the `actor_id`, making it our responsibility to reference users properly.